README.md

```diff
@@ -1,4 +1,34 @@
-# Sortable
+# Admin Sortable
+Admin Sortable is a Laravel package for Craftable used to add sortable feature for you Eloquent models.
+
+## Requirements
+Admin Sortable requires:
+1. php 7.1+
```

Comment:
tu mi mozno napada otazka ci to nie je lepsie dat tu aj v composer.json na rovnaku verziu php ako ma craftable ("^7.4.0|^8.0") z dovodu, ze routes sa generuju podla "novej struktury", ktoru sme releasovali v tomto PR (https://github.com/BRACKETS-by-TRIAD/admin-generator/pull/52/files), ale zaroven to asi nie je breaking ani pre starsie verzie craftable